libsolidity/codegen/ir/IRGenerationContext.h

```diff
@@ -65,13 +65,17 @@ using InternalDispatchMap = std::map<YulArity, DispatchSet>;
 class IRGenerationContext
 {
 public:
+	enum class ExecutionContext { Creation, Runtime };
```

Comment:
```suggestion
	enum class ExecutionContext { Creation, Deployed };
```
I know we use "runtime" a lot, but it is not correct - runtime is the opposite of compile-time.